docs/self-managed/setup/deploy/amazon/amazon-eks/eksctl.md

```diff
@@ -574,39 +569,415 @@ eksctl create iamserviceaccount \
+
+```bash
+kubectl get secret setup-db-secret -o yaml --namespace camunda
+```
+
+This should display the secret with the base64 encoded values.
+
+3. **Create a copy of the manifest**: Save the above manifest to a file, for example, `setup-postgres-create-db.yml`.
+
+```yaml reference
+https://github.com/camunda/camunda-tf-eks-module/blob/feature/opensearch-doc/examples/camunda-8.6/setup-postgres-create-db.yml
+```
+
+3. **Apply the manifest**: Once the secret is created, the **Job** manifest from the previous step can consume this secret to securely access the database credentials.
+
+```bash
+kubectl apply -f setup-postgres-create-db.yml --namespace camunda
 ```
+
+4. **Verify the job's completion**: Once the job is created, you can monitor its progress using:
+
+```bash
+kubectl get job/create-setup-user-db --namespace camunda --watch
+```
+
+Once the job shows as `Completed`, the users and databases will have been successfully created.
+
+5. **Check logs for confirmation**: You can view the logs of the job to confirm that the users were created and privileges were granted successfully:
+
+```bash
+kubectl logs job/create-setup-user-db --namespace camunda
+```
+
+6. **Cleanup the resources:**
+
+```bash
+kubectl delete job create-setup-user-db --namespace camunda
+kubectl delete secret setup-db-secret --namespace camunda
+```
+
+By running these commands, you will clean up both the job and the secret, ensuring that no unnecessary resources remain in the cluster.
+
+## 4. OpenSearch domain
+
+Creating an OpenSearch domain can be accomplished through various methods, such as using the AWS Management Console or the AWS CLI. This guide focuses on providing a reproducible setup using the CLI. For information on creating an OpenSearch domain using the UI, refer to the [AWS OpenSearch documentation](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/create-managed-domain.html).
+
+The resulting OpenSearch domain is intended for use with the Camunda platform, the following components utilize OpenSearch:
+
+- Zeebe
+- Operate
+- Tasklist
+- Optimize
+
+:::info Optional service
+
+If you don't want to use the Amazon OpenSearch managed service for OpenSearch, you can skip this section.
+However, please note that you may need to adjust the following instructions to remove references to it.
+
+If you choose not to use this service, you'll need to either provide a managed OpenSearch or Elasticsearch service or use the internal deployment by the Camunda Helm chart in Kubernetes.
+
+:::
+
+:::note Available since Camunda 8.4
+
+As of the 8.4 release, Zeebe, Operate, and Tasklist are now compatible with [Amazon OpenSearch Service](https://aws.amazon.com/de/opensearch-service/) 2.5.x. Note that using Amazon OpenSearch Service requires [setting up a new Camunda installation](/self-managed/setup/overview.md). A migration from previous versions or Elasticsearch environments is currently not supported.
+
+:::
```

Comment:
Does this note still make sense, considering we're listing couples lines above the ones that work with OpenSearch / Elasticsearch?

Reply:
Maybe the note could just be generalised to something along the lines of

> It is not feasible to interchangeably switch between Elasticsearch and OpenSearch post Camunda deployment.